### PR TITLE
fix gulp gh-pages task to work issue #41

### PR DESCRIPTION
**Pull request by iamJoeTaylor**
cc @square/field-kit-contributors

### CODE REVIEW

**Comment by eventualbuddha** [comment]
Rebase?

**Comment by iamJoeTaylor** [comment]
Rebased.

File: public/javascript/field-kit.js [hunk 1]
```diff
@@ -2889,84 +2898,26 @@ var DelimitedTextFormatter = (function (_Formatter) {
 exports['default'] = DelimitedTextFormatter;
 module.exports = exports['default'];
 
-},{"./formatter":14}],11:[function(_dereq_,module,exports){
+},{"./formatter":14}],11:[function(require,module,exports){
```
Comment (eventualbuddha) [inline public/javascript/field-kit.js]: Was this supposed to be `require` instead of `_dereq_`?

Comment (iamJoeTaylor) [inline public/javascript/field-kit.js]: :( yes, fixed.

**Comment by eventualbuddha** [comment]
:+1:

**Pull request merged**